tests/setup/jest.globals.ts

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import type * as core from "express-serve-static-core"
+import { RequestWithJwt } from "components/types/types"
+import { Response } from "express"
+
+function asMock<T>(value: T): jest.Mock {
+    return value as jest.Mock
+}
+
+function mockRequestWithJwt(): RequestWithJwt<core.Query, any> {
+    return <RequestWithJwt<core.Query, any>>{}
+}
+
+function mockResponse(): Response {
+    const respose = <Response>{}
+
+    const mockImplementation = () => {
+        return respose
+    }
+
+    respose.status = jest.fn().mockImplementation(mockImplementation)
+    respose.json = jest.fn()
+    respose.end = jest.fn()
+
+    return respose
+}
+
+function getResolvingPromise<T>(value?: T): Promise<T> {
+    return new Promise((resolve) => {
+        resolve(value)
+    })
+}
+
+function getMockCallArgument<T>(value: any, call: number, argument: number): T {
+    const calls = asMock(value).mock.calls
+
+    expect(calls.length).toBeGreaterThanOrEqual(call)
+    expect(calls[call].length).toBeGreaterThanOrEqual(argument)
+
+    return asMock(value).mock.calls[call][argument] as T
+}
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const g = global as any
+
+g.asMock = asMock
+g.mockRequestWithJwt = mockRequestWithJwt
+g.mockResponse = mockResponse
+g.getResolvingPromise = getResolvingPromise
+g.getMockCallArgument = getMockCallArgument
```

Comment:
Can't we do this as a module like everything else?

Reply:
I moved the functions to a module, but to get them into the global this is the way.